lib/__tests__/build-files.test.js

```diff
@@ -88,10 +88,14 @@ test('Generated table of contents', function() {
 });
 
 test('Concatenated CSS files', function() {
+  let hash = fs.readFileSync(
+    buildDir + '/' + siteConfig.projectName + '/hash.txt',
```

Comment:
What is there a need for `hash.txt`?

Reply:
the hash.txt file is needed in order to supply the CSS hash string to the test suites. but this is only a temporary fix because as I say in my previous comment adding a hash.txt file on the build folder is not the right nor cleanest approach to this. Do you have any suggestion on this?

Reply:
@yangshun